lib/simplifyExpression/basicsSearch/basicRules.js

```diff
@@ -6,22 +6,25 @@ const checks = require('../../checks');
 const defineRuleString = require('../../util/defineRuleString');
 const Node = require('../../node');
 
-// e.g. -(-3) -> 3
-const NEGATION = defineRuleString('--#a', '#a');
-
-// Simplifies two unary minuses in a row by removing both of them.
-// e.g. -(- 4) --> 4
-function negation(node) {
-  const newNode = rewriteNode(NEGATION, node);
+// Helper function to apply a rule or its reverse to the inputNode
```

Comment:
I was thinking to move it into a separate file as a `util`. Where do you think I should put it?